patch-upstream.sh

```diff
@@ -17,6 +17,9 @@ cp -rf adot/* opentelemetry-lambda/
 # collector used in each Lambda layer
 cd opentelemetry-lambda/collector
 
+# patch otel version on collector/go.mod
+patch < ../../OTEL_Version.patch
```

Comment:
don't we need a -p2 here? because you will be inside the collector folder. -p2 will make you ignore 2 leading slashes.

Reply:
It did work with -p2 here but it makes sense. Updated now